web/satellite/src/utils/doNotTrack.ts

```diff
@@ -0,0 +1,15 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+export function isDoNotTrackEnabled () {
+    const doNotTrackOption = (
+      window.doNotTrack ||
+      window.navigator.doNotTrack
+    );
+
+    if (!doNotTrackOption) {
+      return false;
+    }
+
+    return doNotTrackOption.charAt(0)  === '1' || doNotTrackOption === 'yes';
```

Comment:
This crash if `donNotTrackOption` is of string type.
You could change the above conditional for avoiding this crash returning with false if it isn't of string type rather than only check for any JS value which evaluates to false.